src/components/contact/assets/contact.scss

```diff
@@ -26,11 +26,11 @@ section.contact {
     text-align: right;
     font-size: $font-xxl;
     color: $green500;
-    font-weight: 300;
+    font-weight: $font-weight-thin;
     text-transform: uppercase;
     border-bottom: 1px solid $gray;
     font-family: $font-fjalla-one;
-    margin-bottom: 0;
+    margin-bottom: $margin-none;
```

Comment:
I don't think we need a variable for margin...

Reply:
ohh i missed it. will push in next push if we have any more changes 👍